packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -19,6 +21,7 @@ import { request } from './utils/request'
 // TODO this is an issue that we cannot share types from the client to other packages
 
 const MAX_RETRIES = 10
+const DEFAULT_REQUEST_TIMEOUT_MS = 30000
```

Comment:
```suggestion
const DEFAULT_REQUEST_TIMEOUT_MS = 30_000
```

Why is the default 30s? I'm curious

Reply:
Note from Slack: Magic number, could be any other value that seems reasonable.